pylops/signalprocessing/Sliding2D.py

```diff
@@ -135,7 +135,7 @@ def Sliding2D(Op, dims, dimsd, nwin, nover, tapertype="hanning", design=False):
 
     combining = HStack(
         [
-            Restriction(dimsd, range(win_in, win_end), dtype=Op.dtype).H
+            Restriction(dimsd, range(win_in, win_end), axis=0, dtype=Op.dtype).H
```

Comment:
My main worry was not breaking any tests and `axis=-1` breaks a few. With regard to the design, I'm happy to keep it as is, since sliding along the slowest does make sense to me.